src/test/java/org/zaproxy/zest/test/v1/ZestClientLaunchUnitTest.java

```diff
@@ -80,11 +87,7 @@ void testHtmlUnitLaunch() throws Exception {
         // runner.setProxy("localhost", 8090);
         runner.run(script, null);
 
-        verifyUrlAccessed(PATH_SERVER_FILE);
-    }
-
-    private void verifyUrlAccessed(String filePath) {
-        server.verify(getRequestedFor(urlMatching(filePath)));
+        assertThat(urlAccessed).isTrue();
```

Comment:
I'm not disagreeing with this. I'm just wondering if we should be settling on a preferred format/methodology (across ZAP's repos).

`assertTrue`
`is(equalTo(true))`
`isTrue()`
`.isEqualTo(true)`
[I'm assuming those are all syntactically correct, I haven't double checked...but you get the idea]

(For discussion/thought, doesn't necessarily need change here)